ci/src/mainnet_revisions/mainnet_revisions.py

```diff
@@ -137,15 +141,20 @@ def get_subnet_replica_version_info(subnet_id: str) -> (str, str):
 
     version = response["payload"]["replica_version_to_elect"]
     hash = response["payload"]["release_package_sha256_hex"]
+    launch_measurements = response["payload"]["guest_launch_measurements"]
 
     dev_hash = download_and_hash_file(
         f"https://download.dfinity.systems/ic/{version}/guest-os/update-img-dev/update-img.tar.zst"
     )
 
-    return (version, hash, dev_hash)
+    dev_measurements = download_and_read_file(
+        f"https://download.dfinity.systems/ic/{version}/guest-os/update-img-dev/launch-measurements.json"
+    )
+
+    return (version, hash, dev_hash, launch_measurements, dev_measurements)
 
 
-def get_latest_replica_version_info() -> (str, str):
+def get_latest_replica_version_info() -> (str, str, str, str, str):
```

Comment:
It would be less error prone if this function and `get_latest_hostos_version_info()` below start returning a structure / dictionary instead of a long tuple of strings.